CAM/SetCamFarDof.md

```diff
@@ -8,6 +8,7 @@ ns: CAM
 void SET_CAM_FAR_DOF(Cam cam, float farDOF);
 ```
 
+Seems to specify when the camera should stop being in focus. (Appears to be in standard distance units)
```

Comment:
`(Appears to be in standard distance units)` should just be `farDOF is in standard units`

Reply:
Fixed